CHANGELOG.md

```diff
@@ -8,6 +8,76 @@
 
 <!-- next version -->
 
+## v1.29.0
+
+### 🛑 Breaking changes 🛑
+
+- `vcs`: Add the VCS metrics inspired by the GitHub Receiver (#1372)
+  Makes the following changes:
+
+  - Add metrics `vcs.change.count`, `vcs.change.duration`, `vcs.change.time_to_approval`, `vcs.repository.count`, `vcs.ref.count`,
+    `vcs.ref.lines_delta`, `vcs.ref.revisions_delta`, `vcs.ref.time`, `vcs.contributor.count`
+  - The VCS attributes `vcs.change.state`, `vcs.revision_delta.direction` and `vcs.line_change.type` have been added to the registry.
+  - The VCS ref attributes have been duplicated to `vcs.ref.base.*` to allow for ref comparisons.
+  - The VCS attribute `vcs.ref.type` has been added for simplicity when neither a full head or base ref is necessary.
+  - `vcs.repository.change.*` attributes have been deprecated and moved to `vcs.change.*`.
+  - `vcs.repository.ref.*` attributes have been deprecated and moved to `vcs.ref.head.*`.
+
+- `feature_flag`: Rename `feature_flag` event to `feature_flag.evaluation` event, define new feature flag attributes and provide body definition. Remove `feature_flag` span event definition in favor of log-based event.
+ (#1440)
```

Comment:
unrelated to this PR, it would be nice if these were links

Reply:
absolutely!

https://github.com/open-telemetry/semantic-conventions/issues/1627